docs/aurora/data-management/daos/daos-overview.md

```diff
@@ -1,3 +1,267 @@
+# DAOS Architecture
+DAOS is a high performance storage system for storing checkpoints and analysis files.
+DAOS is fully integrated with the wider Aurora compute fabric as can be seen in the overall storage architecture below.
+
+![Aurora Storage Architecture](aurora-storage-architecture.png "Aurora Storage Architecture")
+
 # DAOS Overview
+Users should submit a request as noted below to have a DAOS pool created for their project.
+Once created, users may create and manage containers within the pool as they wish.
+
+
+### Note
+This is an initial test DAOS configuration and as such, any data on the DAOS system will envtually be deleted when
+the configuration is changed into a larger system.
+Warning will be given before the system is wiped to allow time for users to move any important data off.
+
+## Pool Allocation
+Email support@alcf.anl.gov to request a pool and provide which primary project you're on.
+The pool will be set to allow anyone in the project unix group to access the pool.
+Please request the capacity of allocation you would like.
+
+## Modules
+Please load the daos/base module when using DAOS.
+This should be done when logging into the UAN or when using DAOS from a compute job script.
+
+`module load daos/base`
+
+## Pool
+Confirm you are able to query the pool.
+
+`daos pool query <pool name>`
+
+Example output
+> daos pool query software  
+> Pool 050b20a3-3fcc-499b-a6cf-07d4b80b04fd, ntarget=640, disabled=0, leader=2, version=131  
+> Pool space info:  
+> - Target(VOS) count:640  
+> - Storage tier 0 (SCM):  
+> Total size: 6.0 TB  
+>   Free: 4.4 TB, min:6.5 GB, max:7.0 GB, mean:6.9 GB  
+> - Storage tier 1 (NVMe):  
+>   Total size: 200 TB  
+>   Free: 194 TB, min:244 GB, max:308 GB, mean:303 GB  
+> Rebuild done, 4 objs, 0 recs
+
+## Container
+The container is the basic unit of storage.
+A POSIX container can contain hundres of millions of files, you can use it to store all of your date.
```

Comment:
spelling: "hundreds of millions" & "your data"

Reply:
corrected